mackerel-plugin-mssql/lib/mssql_windows.go

```diff
@@ -0,0 +1,183 @@
+// +build windows
+
+package mpmssql
+
+//go:generate wmi2struct -n -p mpmssql -o wmi.go Win32_PerfRawData_MSSQLSQLEXPRESS_MSSQLSQLEXPRESSBufferManager Win32_PerfRawData_MSSQLSQLEXPRESS_MSSQLSQLEXPRESSSQLStatistics Win32_PerfRawData_MSSQLSQLEXPRESS_MSSQLSQLEXPRESSGeneralStatistics Win32_PerfRawData_MSSQLSQLEXPRESS_MSSQLSQLEXPRESSAccessMethods
```

Comment:
Which environment does this `go generate` require?
Apparently it needs some Windows and https://github.com/mattn/wmi2struct installed, but if there're other requirements (Windows version, SQL Server installed, …) please refer it in somewhere like README.

Reply:
No need to do `go generate` for build. This is a way to update wmi structs when a bug of wmi2struct is found. For example, type convertion might be wrong, or yet another types must be listed.

https://github.com/mattn/wmi2struct/blob/db7bf4ef341acaef52b9b7bb11c6f5d8e3dee52a/main.go#L32

Reply:
I want to avoid the case that someone will want to re-generate it for some reason and they don't understand what is necessary for `generat`ing.  So, I still think it's better to be documented wmi2struct (and SQL Server ?) is necessary for this `go generate`.